dot/sync/chain_sync_integeration_test.go

```diff
@@ -205,34 +168,34 @@ func TestChainSync_sync_tip(t *testing.T) {
 	require.Equal(t, tip, cs.state)
 }
 
-func TestChainSync_getTarget(t *testing.T) {
+func TestChainSync_getTarget_Integration(t *testing.T) {
```

Comment:
This is not an integration test, please move back to `chain_sync_test.go`

Reply:
Moved to chain_sync_test.go, and renamed.